spring-boot-modules/spring-boot-swagger-jwt/src/main/java/com/baeldung/swaggerjwt/configuration/SwaggerConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.swaggerjwt.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.*;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.service.contexts.SecurityContext;
+import springfox.documentation.spring.web.plugins.Docket;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Configuration
+public class SwaggerConfiguration {
+
+    public static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private ApiInfo apiInfo() {
+        return new ApiInfo("My REST API",
+		  "Some custom description of API.",
+		  "1.0",
+		  "Terms of service",
+		  new Contact("Sallo Szrajbman",
+		    "www.baeldung.com",
+		    "salloszraj@gmail.com"),
```

Comment:
The new Contact(...) can still be on one line; it's just the lengthy list of parameters passed to the ApiInfo constructor that won't fit on one line and need to be broken up.

Reply:
Thanks for the heads up! It's now fixed.